pkg/identificationtypes/client.go

```diff
@@ -0,0 +1,37 @@
+package identificationtypes
```

Comment:
nome do pacote deve seguir padrão dos outros recursos, no singular
```suggestion
package identificationtype
```

Reply:
Feito.